docs/2.0.0-Beta/litmus-namespaced-2.0.0-Beta.yaml

```diff
@@ -0,0 +1,288 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: litmus-portal-admin-config
+data:
+  AgentScope: namespace
+  AgentNamespace: ${LITMUS_PORTAL_NAMESPACE}
+  DataBaseServer: "mongodb://mongo-service:27017"
+  JWTSecret: "litmus-portal@123"
+  DB_USER: "admin"
+  DB_PASSWORD: "1234"
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: litmusportal-frontend
+  labels:
+    component: litmusportal-frontend
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      component: litmusportal-frontend
+  template:
+    metadata:
+      labels:
+        component: litmusportal-frontend
+    spec:
+      containers:
+        - name: litmusportal-frontend
+          image: litmuschaos/litmusportal-frontend:2.0.0-Beta1
+          resources:
+            limits:
+              cpu: 200m
+              memory: 400Mi
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8080
+          env:
+            - name: AGENT_SCOPE
+              valueFrom:
+                configMapKeyRef:
+                  name: litmus-portal-admin-config
+                  key: AgentScope
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: litmusportal-frontend-service
+spec:
+  type: NodePort
+  ports:
+    - name: http
+      port: 9091
+      targetPort: 8080
+  selector:
+    component: litmusportal-frontend
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: litmus-server-account
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: litmus-server
+  labels:
+    name: litmus-server
+rules:
+  - apiGroups:
+      - "*"
```

Comment:
Why is this `*/*/*`

Reply:
@ksatchit  this require permission to create role and rolebindings and it should have all the permission that is inside the rbac we are creating so eventually it becomes very large and complex so we gave all and checking the upper limit from other rbac. We do same thing in cluster mode as well.